repository/spark/tests/spark-job/spark-pi.yaml

```diff
@@ -0,0 +1,31 @@
+apiVersion: "sparkoperator.k8s.io/v1beta2"
+kind: SparkApplication
+metadata:
+  name: spark-pi
+  namespace: spark
+spec:
+  type: Scala
+  mode: cluster
+  image: "mesosphere/spark:spark-2.4.5-hadoop-2.9-k8s"
+  imagePullPolicy: Always
+  mainClass: org.apache.spark.examples.SparkPi
+  mainApplicationFile: "local:///opt/spark/examples/jars/spark-examples_2.11-2.4.5.jar"
+  arguments:
+    - "1000"
+  sparkConf:
+    "spark.ui.port": "4041"
```

Comment:
`spark.ui.port` is not necessary here, probably `arguments` can be dropped as well

Reply:
yeah... dropping this makes sense... part of providing this as a separate file was to provide an example to users.  It is primarily for the test of course.   I think I leave the args (realizing that they are not necessary).  It does confirm that it takes arguments... and it will fail if that changes... which is somewhat of a test that provides awareness.